villageline/settings/local.py.example

```diff
@@ -0,0 +1,8 @@
+from settings import *
+
+# SECRET_KEY =
+# ALLOWED_HOSTS = []
+# TWILIO_ACCOUNT_SID = ""
+# TWILIO_AUTH_TOKEN = ""
+# SENDGRID_API_KEY = ""
+# PYTHONBREAKPOINT = "IPython.embed"
```

Comment:
I don't think this is needed / helpful in general - I had it in the local environment file I mentioned to you, but that's just because I prefer using it to debug rather than `pdb`.